validation-framework/build.sh

```diff
@@ -10,16 +10,30 @@ chartver=""
 if [ "$2" ];then
   chartver="$2"
 fi
-
+composeCMD="docker-compose"
+containerDelim="-"
 opt3=$3
-
+function compose {
+  if [ "$composeCMD" = "podman-compose" ]; then
+    podman-compose $@
+  else
+    docker compose $@
+  fi
+}
 function preCheck {
   if ! command -v docker compose &> /dev/null
```

Comment:
this command is actually trying to test if both docker and compose command were installed. Once this is changed back, the containerDelim won't be set correct, and the validation step will fail.